contracts/Registry.sol

```diff
@@ -193,6 +194,8 @@ contract Registry {
         challenges[challengeNonce].listingHash = _listingHash;
         listing.challengeID = challengeNonce;
 
+        token.transfer(challengeAddress, minDeposit);
```

Comment:
@emaG3m may want to wrap this in a `require`. Also, might make sense to move the `transferFrom` line directly above this one, just to group the value transferring logic together